presentations/en/2019-11-05-CIOEC-Mandate.md

```diff
@@ -159,6 +168,7 @@ Revisit Investment Management practices
 - Promote hiring of those who demonstrate curiosity, adaptability, and ethical behaviour
 
 Note:
-Monitor investments on business outcomes, not project plans
+Significant impact on our relationship with CFOB and lines of businesses
+Work with CFOB to assure our fiduciary responsibilities to public funds are still fulfilled
```

Comment:
Do we need to say this? We would like to ensure we meet all of our responsibilities from all partners? Seems weird to specifically list finance here? Is there a reason particular focus is paid to them ... ?

Reply:
This is the section on re-visiting investment management practices. So finance is a key stakeholder. The way finance is monitored is currently not outcome-based, it's task-based.
The two quoted lines here are part of the Notes section to engage in conversation with CIOEC audience if they want more details.